### PR TITLE
AHTI-201 | Import and serve harbor specific details from venepaikka

### PR DESCRIPTION
**Pull request by charn**
Harbor specific information is imported from venepaikka API. Imported data is processed into a structured JSON blob and saved to `FeatureDetails.data`. This data is then served from the ahti GQL API.

**Note**
Currently the venepaikka API doesn't provide depth information for berths. In the tests I assumed that the information might be available at some point.

### CODE REVIEW

File: features/importers/venepaikka_harbors/importer.py [hunk 1]
```diff
@@ -198,6 +226,24 @@ def _import_service_map_url(self, feature: Feature, servicemap_id: str):
...
 
+    @staticmethod
+    def _import_harbor_details(feature: Feature, harbor_details: Mapping):
+        data = {}
+
+        if harbor_details["berth_moorings"]:
+            data["berth_moorings"] = list(set(harbor_details["berth_moorings"]))
+
+        if harbor_details["berth_depths"]:
+            data["berth_min_depth"] = min(harbor_details["berth_depths"])
```
Comment (igordavydsson) [inline features/importers/venepaikka_harbors/importer.py:237]: well, we do not know when these will be added. probably when we have a UI for harbors personnel, they might start adding those for some berths gradually. even the, I am pretty sure, there will be a very long period when some of the berths will have `null` depth, while other might have some value. not sure how this affects ahti in general, but this importer probably filters out `null`s here `berth_depths: properties.piers.edges[*].node.properties.berths.edges[*].node.berthType.depth[]`, right?

Comment (charn) [reply features/importers/venepaikka_harbors/importer.py:237]: Yeah, null values are ignored. Looks like the jmespath tool automatically ignores them.

Comment (charn) [reply features/importers/venepaikka_harbors/importer.py:237]: I added one berth with depth information into the faked/test response, but all others have depth as `null`.

Comment (igordavydsson) [reply features/importers/venepaikka_harbors/importer.py:237]: > jmespath tool seems to automatically ignore them
yeah, that's how I understood it 👍 

alright then!

File: features/importers/venepaikka_harbors/importer.py [hunk 2]
```diff
@@ -32,6 +32,29 @@
           phone
           email
           servicemapId
+          maximumWidth
+          maximumLength
+          maximumDepth
```
Comment (igordavydsson) [inline features/importers/venepaikka_harbors/importer.py:37]: these will be renames to `max<Width|Length|Depth>` in the next couple of days. just as a heads up 😅

**Comment by dankiki** [comment]
all good, only commits need squashing.

**Pull request merged**